docs/source-fabric/guide/multi_node/cloud.rst

```diff
@@ -35,66 +29,107 @@ A page opens in your browser where you can follow the instructions to complete t
+            optimizer = torch.optim.SGD(model.parameters(), lr=0.1)
 
-            # Your training code
-            model = ...
-            optimizer = ...
             model, optimizer = fabric.setup(model, optimizer)
-            ...
+            train_dataloader = fabric.setup_dataloaders(train_dataloader)
+
+            for batch_idx, batch in enumerate(train_dataloader):
+                input, target = batch
+                output = model(input, target)
+                loss = F.nll_loss(output, target.view(-1))
+                fabric.backward(loss)
+                optimizer.step()
+                optimizer.zero_grad()
+
+                if batch_idx % 10 == 0:
+                    fabric.print(f"iteration: {batch_idx} - loss {loss.item():.4f}")
+
+
+        if __name__ == "__main__":
+            main()
+
+|
 
-**Step 2:** Init a ``lightning.app.core.app.LightningApp`` with the ``FabricMultiNode`` component.
-Configure the number of nodes, the number of GPUs per node, and the type of GPU:
+**Step 3:** Remove hardcoded accelerator settings if any and let Lightning automatically set them for you. No other changes are required in your script.
 
 .. code-block:: python
-    :emphasize-lines: 5,7
-    :caption: app.py
 
-    # 2. Create the app with the FabricMultiNode component inside
-    app = L.LightningApp(
-        FabricMultiNode(
-            MyTrainingComponent,
-            # Run with 2 nodes
-            num_nodes=2,
-            # Each with 4 x V100 GPUs, total 8 GPUs
-            cloud_compute=L.CloudCompute("gpu-fast-multi"),
-        )
-    )
+    # These are the defaults
+    fabric = L.Fabric(accelerator="auto", devices="auto")
 
+    # DON'T hardcode these, leave them default/auto
+    # fabric = L.Fabric(accelerator="cpu", devices=3)
 
-**Step 3:** Run your code from the CLI:
+|
 
-.. code-block:: bash
+**Step 4:** Install dependencies and download all necessary data. Test that your script runs in the studio first. If it runs in the studio, it will run in multi-node!
 
-    lightning run app app.py --cloud
+|
 
-This command will upload your Python file and then opens the app admin view, where you can see the logs of what's happening.
+**Step 5:** Open the Multi-Machine Training (MMT) app. Type the command to run your script, select the machine type and how many machines you want to launch on. Click "Run" to start the job.
 
-.. figure:: https://pl-public-data.s3.amazonaws.com/assets_lightning/fabric/fabric-multi-node-admin.png
-   :alt: The Lightning AI admin page of an app running a multi-node fabric training script
-   :width: 100%
+.. video:: https://pl-public-data.s3.amazonaws.com/assets_lightning/fabric/videos/lightning-ai-mmt-demo-fabric.mp4
+    :width: 800
+    :loop:
+    :muted:
+
+After submitting the job, you will be redirected to a page where you can monitor the machine metrics and logs in real-time.
 
 
 ----
 
 
+****************************
+Bring your own cloud account
+****************************
```

Comment:
I felt for this was relevant for this audience, but I may not have described it very well. If you have suggestions please let me know.